README.md

```diff
@@ -1,2 +1,130 @@
-# content-event-current-day
-HTML Theme displaying current and next event using the `rise-google-calendar` web component.
+## Introduction
+
+This HTML theme was created to display the current and next event for a single day using data from a google calendar using the `rise-google-calendar` component. Full documentation for the component can be found[here.](http://rise-vision.github.io/rise-google-calendar/components/rise-google-calendar/) 
+
+
+## Steps to run the Current Event theme
+
+1. Fork or download the entire content-event-current-day repository. You will also need to run `npm install` and `bower install` to install the dependencies required for the the template. Bower is a package manager for Javascript libraries and frameworks. All third-party Javascript dependencies are listed in the bower.json file. To install Bower, run the following command in Terminal: `npm install -g bower`. Should you encounter any errors, try running the following command instead: `sudo npm install -g bower`.
```

Comment:
@pcsandford *dependencies required for the the template* should read *dependencies required for the the theme*